src/Native/Runtime/MathHelpers.cpp

```diff
@@ -82,6 +82,70 @@ EXTERN_C REDHAWK_API UInt64 REDHAWK_CALLCONV RhpDbl2ULng(double val)
     return((UInt64)val);
 }
 
+// CORERT Specific - on Project N the arguments to these helpers are inverted
+#ifdef CORERT
+#include <cmath>
+
+#define CLR_NAN_32 0xFFC00000
+#define CLR_NAN_64 0xFFF8000000000000L
```

Comment:
(Not sure how portable this is - should I make it use one of the silent NaN C++ constructors?)